src/file.ts

```diff
@@ -1120,6 +1121,10 @@ class File extends ServiceObject<File> {
    *     NOTE: Byte ranges are inclusive; that is, `options.start = 0` and
    *     `options.end = 999` represent the first 1000 bytes in a file or object.
    *     NOTE: when specifying a byte range, data integrity is not available.
+   * @property {boolean} [decompress=true] Disable auto decompression of the
```

Comment:
Is this from the perspective of the GCS service?

Reply:
No, nodejs-sotage API always requests that the data should be served as is (compressed if applicable) i. e. sending appropriate [headers](https://github.com/googleapis/nodejs-storage/blob/b2660d8c7d68058634a20ae923afe6a44abe6b4e/src/file.ts#L1239-L1242)
```ts
const headers = {
  'Accept-Encoding': 'gzip',
  'Cache-Control': 'no-store',
} as Headers;
```

This flag is only from the perspective of nodejs-storage client to allow this library to not transcode the data if received from the server in compressed form and return to user compressed..

Reply:
@frankyn ping!